packages/admin-app/src/components/ElectionSetupWizard/ElectionSetupWizard.tsx

```diff
@@ -46,10 +49,14 @@ export const ElectionSetupWizard: React.FC = () => {
         };
         setRequest(newRequest);
     };
-
-    const handleSubmit = () => {
+    const handleUploadManifest = (manifestJson: ValidateManifestRequest) => {
+        setManifest(manifestJson);
+    };
+    const handleSubmit = async () => {
+        const v1Client = ClientFactory.GetV1Client();
+        await v1Client.manifestPut(manifest);
```

Comment:
For testing purposes I tried submitting the uploaded manifest through the nswag API and it worked, so I left this code in for now because I'll need it in the next PR anyway